actions/build-push-to-dockerhub/action.yaml

```diff
@@ -25,15 +25,36 @@ inputs:
     description: |
       The dockerfile to use.
     required: false
+  build-args:
+    description: |
+      List of arguments necessary for the Docker image to be built.
+    required: false
+    default: ""
+  target:
+    description: |
+      Target stage to build
+    required: false
+  cache-from:
+    description: |
+      Where cache should be fetched from
+    required: false
+    default: "type=gha"
+  cache-to:
+    description: |
+      Where cache should be stored to
+    required: false
+    default: "type=gha,mode=max"
 
 runs:
   using: composite
   steps:
+    # See this conversation for more context as to why we don't want to allow pushes on pull requests
+    # https://github.com/grafana/shared-workflows/pull/143#discussion_r1628314620
     - name: Check if push is allowed
-      if: ${{ inputs.push == 'true' && github.event_name != 'push' }}
+      if: ${{ inputs.push == 'true' && github.event_name != 'pull_request' }}
```

Comment:
did you mean this?
```suggestion
      if: ${{ inputs.push == 'true' && github.event_name == 'pull_request' }}
```

Reply:
ooooh that's for the check - looks like @Elfo404 is right. so if push input is true and is also a pull_request we should exit. thanks @Elfo404, dismissing the review for now

Reply:
ah yes, you're right! good catch. updated 😄